### PR TITLE
:sparkles: Show blob divert event

### PR DESCRIPTION
**Pull request by foysalit**
This is dependent on https://github.com/bluesky-social/atproto/pull/2238

This PR also adjust label SDK usage within ozone-ui to align with latest `main` on atproto repo.

### CODE REVIEW

File: components/common/labels/Grid.tsx [hunk 1]
```diff
@@ -28,19 +22,10 @@ export const LabelSelector = (props: LabelsProps) => {
...
-        const labelText = typeof label === 'string' ? label : label.id
-        return {
-          label: labelText,
-          value: labelText,
-        }
-      }),
-    }),
-  )
+  const selectorOptions = Object.values(ALL_LABELS).map((labelOption) => ({
+    label: labelOption.identifier,
```
Comment (devinivy) [inline components/common/labels/Grid.tsx:26]: Any idea what this new type error is?

**Review by devinivy — APPROVED** [review]
Wasn't positive about the new handling of labels, but all the divert stuff looks good 👍

File: components/common/labels/Grid.tsx (hunk 1 above)
Comment (devinivy) [reply components/common/labels/Grid.tsx:26]: (I assume this is related to @atproto/api updates.)

Comment (foysalit) [reply components/common/labels/Grid.tsx:26]: yep, cleaning it up now. I was referencing a local version of the api package so had to adjust things to get the app to run.

**Comment by bnewbold** [comment]
I expected a "Divert" action as an option in the quick-action menu for posts with an image attached, but don't see that in the current render preview?

**Comment by foysalit** [comment]
> I expected a "Divert" action as an option in the quick-action menu for posts with an image attached, but don't see that in the current render preview?

ahh classic useEffect bug. pushed out the fix. mind giving it another go @bnewbold ?

**Pull request merged**